examples/kubernetes/otel-collector.yaml

```diff
@@ -64,20 +64,57 @@ data:
               k8s.pod.name: 'EXPR($.pod_name)'
               k8s.container.restart_count: 'EXPR($.restart_count)'
               k8s.pod.uid: 'EXPR($.uid)'
+          # tag the resource level attribute in metadata
+            resource:
+              k8s.pod.uid: 'EXPR($.uid)'
           # Clean up log body
           - type: restructure
             id: clean-up-log-body
             ops:
               - move:
                   from: log
                   to: $
+                  
+                  
+    processors:
+      # k8sattributes processor to get the metadata from K8s
+      k8sattributes:
+        auth_type: "serviceAccount"
+        passthrough: false
+        extract:
+          metadata:
+            - k8s.pod.name
+            - k8s.pod.uid
+            - k8s.deployment.name
+            - k8s.cluster.name
+            - k8s.namespace.name
+            - k8s.node.name
+            - k8s.pod.start_time
+          # Pod labels which can be fetched via K8sattributeprocessor
+          labels:
+            - tag_name: k8s.label.app
+              key: app
+              from: pod
+            - tag_name: k8s.label.component
+              key: component
+              from: pod
+        # Pod association using resource attributes and connection
+        pod_association:
+          - from: resource_attribute
+            name: k8s.pod.uid
+          - from: resource_attribute
+            name: k8s.pod.ip
+          - from: connection
+            name: ip
```

Comment:
I don't think `connection` uses `name` field

Reply:
BTW, we might want to improve the processor docs. I thought that keeping "name" there was used as a convention :)

Reply:
yeah, I missed looking at the code of this pod association. I was not using it as is just kept it consistent. 
Lets update this one as well correctly